transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClientImpl.kt

```diff
@@ -596,3 +611,6 @@ private fun KProperty0<DeploymentConfig?>.isAutostartEnabled(): Boolean =
 
 private fun KProperty0<DeploymentConfig?>.isShowUserTypingEnabled(): Boolean =
     this.get()?.messenger?.apps?.conversations?.showUserTypingIndicator == true
+
+private fun KProperty0<DeploymentConfig?>.isClearConversationEnabled(): Boolean =
+    this.get()?.messenger?.apps?.conversations?.conversationClear?.enabled == true
```

Comment:
according to the charter this feature is disabled by default

Reply:
Again. Wrong charter.